styles.css

```diff
@@ -1,8 +1,8 @@
 .shoelogo { height: 25px; width: 50px; padding-left: 2%; }
-.shoe1 { height: 10px; width: 10px; }
-.shoe2 { height: 10px; width: 15px; }
-.shoe3 { height: 10px; width: 15px; }
-.shoe4 { height: 10px; width: 10px; }
+.shoe1 { height: 100px; width: 100px; }
+.shoe2 { height: 100px; width: 150px; }
+.shoe3 { height: 100px; width: 150px; }
+.shoe4 { height: 100px; width: 100px; }
```

Comment:
Your smaller sizes worked fine, but they were too small to see on the page! 🔍 

I made each image ten times bigger so that you can see how the sizes changed, but also see the shoes when you open your webpage.